### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by xsoulp**
new instructions to run the backend python module from sources

- [ ] Make sure you are opening from a **topic/feature/bugfix branch**
- [ ] Ensure that the PR title represents the desired changes
- [ ] Ensure that the PR description detail the desired changes
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

[^note]:
    Put an `x` into the [ ] to show you have filled the information.
    The template comes from https://github.com/MOV-AI/.github/blob/master/.github/pull_request_template.md
    You can override it by creating .github/pull_request_template.md  in your own repository

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,15 +71,3 @@ For ROS noetic distribution :
 
     docker run -t backend:noetic
 
-## Development stack
```
Comment (engdorm) [inline README.md:67]: why did you delete this?

**Comment by AlexFernandes-MOVAI** [comment]
outdated
